src/common/components/Navbar/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { cloneElement } from 'react';
+import PropTypes from 'prop-types';
+
+import ResponsiveNavigation from '../../../packages/responsive-navigation';
+
+import { Icon, Popover, Menu } from '../';
+
+const Navbar = ({ children }) => (
+  <ResponsiveNavigation>
+    {({ activePath, isMobile, hideNavigation, showNavigation, isNavigationVisible }) => {
+      const Menu = cloneElement(children, { activePath, isMobile });
+
+      return isMobile ? (
+        <Popover
+          content={Menu}
+          onVisibleChange={visible => (visible ? showNavigation() : hideNavigation())}
+          title={<Icon type="close" onClick={hideNavigation} />}
+          trigger="click"
+          visible={isNavigationVisible}
+        >
+          <Icon type="bars" style={{ color: '#fff' }} />
```

Comment:
Maybe make the icon a default prop.
The default is a bit ugly, in most cases you want to override it...